keps/sig-node/4540-strict-cpu-reservation/kep.yaml

```diff
@@ -8,20 +8,19 @@ status: implementable
 creation-date: 2024-03-06
 reviewers:
   - "@ffromani"
-  - "@klueska"                                                                          
   - "@swatisehgal"                                                                      
 approvers:
   - "@sig-node-tech-leads"                                                              
```

Comment:
You might have to identify a specific approver here. Perhaps @ffromani ?

Reply:
I can if we don't have better alternatives :)

Reply:
ok, added @ffromani as the approver.